### PR TITLE
(feat) Add sidenav to outpatient dashboard

### PR DESCRIPTION
**Pull request by samuelmale**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

Adds sidebar navigation

## Related Issue

- https://issues.openmrs.org/browse/O3-1122

## Other

*None.*

### CODE REVIEW

File: packages/esm-patient-registration-app/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "license": "MPL-2.0",
   "homepage": "https://github.com/openmrs/openmrs-esm-patient-management#readme",
   "scripts": {
-    "start": "openmrs develop",
+    "start": "openmrs develop --backend https://ohri-demo.globalhealthapp.net",
```
Comment (brandones) [inline packages/esm-patient-registration-app/package.json:11]:
```suggestion
    "start": "openmrs develop",
```

**Review by brandones — CHANGES_REQUESTED** [review]
Looks good! By which I mean, it looks like the code I wrote :laughing:  Which is great, because that means this will be easy to refactor. Thanks for doing this @samuelmale .

Just one change needed, looks like you left some debug code in.

File: packages/esm-patient-registration-app/package.json (hunk 1 above)
Comment (samuelmale) [reply packages/esm-patient-registration-app/package.json:11]: @brandones lol, why I'm this careless? Nice catch gonna remove that.

Comment (samuelmale) [reply packages/esm-patient-registration-app/package.json]: @brandones resolved.

**Review by pirupius — APPROVED** [review]

**Pull request merged**